docs/developer-guide/mocks-ut-faq.md

```diff
@@ -55,9 +55,11 @@ of the same package as where the interface is defined.
         `docker run -v $PWD:/src -w /src vektra/mockery --all --dir vendor/k8s.io/utils/exec --output pkg/testing/mocks/k8s.io/utils/exec`
     
 ### How to regenerate all existing mocks when interfaces (locally defined or in vendor libraries) are updated?
+    - Execute `make -C go-controller mocksgen`
 
-    - Execute the ```go-controller/hack/regenerate_vendor_mocks.sh``` in situations where all existing mocks have to be regenerated.
-    NOTE: It would take a while(approx 20+ minutes) for all mocks to be regenerated.
+### How to add new pods
```

Comment:
I don't follow this header name. It seems completely out of place - for sure I'm missing something.

Reply:
haha +1 to miguel's point, probably a copy-paste snafu?